### ISSUE TITLE
Broken link in installation doc

### ISSUE THREAD

**Issue by jyu00** [Qiskit/qiskit-metapackage#1016]
## Did you find a doc bug or broken link that needs to be fixed?

Broken link in https://qiskit.org/documentation/install.html#access-ibm-quantum-systems

The bottom of the section has a link to `The IBM Quantum Account`, which is no longer valid as the tutorials have been moved to `https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/docs/tutorials/1_the_ibmq_account.ipynb`

## Does existing documentation need to be updated with new content?

The bullet point
```
4. Run the following commands to store your API token locally for later use in a configuration file called qiskitrc. Replace MY_API_TOKEN with the API token value that you stored in your text editor.
```
should clarify where `qiskitrc` is.

**Comment by garrison** [comment]
I believe that the broken links were fixed in #1023.  The remaining issue about `qiskitrc` is a duplicate of #800, and so I believe the current issue can be closed.

**Comment by jakelishman** [comment]
#1023 was closed without merging, but the whole of `install.rst` was refactored into `getting_started.rst` and `configuration.rst`, and the broken links don't appear to be present any more.  The issue about `qiskitrc` has staled as well, since the file is now called `~/.qiskit/settings.conf`, which is documented in `configuration.rst` (noted in #800).
